### DISCUSSION TITLE
类型替换怎么使用？

### DISCUSSION THREAD

**Discussion by Ador-able** [metatube-community/jellyfin-plugin-metatube#23] [Q&A]
定义好的映射怎么才能生效呢?

**Comment by xjasonlyu** [comment]
等下个版本发布即可。

**Reply by Ador-able** [reply]
[GIN] 2022/06/29 - 03:15:01 | 404 |        40.4µs |      172.17.0.1 | GET      "/v1/actors/search?q=%E5%B0%8F%E8%A5%BF%E3%83%AC%E3%83%8A&provider=GFriends&fallback=False"

[GIN] 2022/06/29 - 03:15:01 | 404 |        40.4µs |      172.17.0.1 | GET      "/v1/actors/search?q=%E9%A3%AF%E7%94%B0%E3%81%9B%E3%81%84%E3%81%93&provider=GFriends&fallback=False"

[GORM] 2022/06/29 03:15:01 github.com/javtube/javtube-sdk-go/engine/movie.go:173

这个演员接口全部都是404错误。这是调用的哪里？可以发出来我连接一下试试么，开着代理，根绝不像是无法访问。

**Reply by xjasonlyu** [reply]
gfriends的
